### PR TITLE
Include NetworkCloud to the CodeOwner file

### PR DESCRIPTION
**Pull request by cn0238**
updating codeowner file

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -734,6 +734,9 @@
 # ServiceLabel: %Network - Private Link
 # ServiceOwners:                                                   @privlinksuppgithub
 
+# PRLabel: %Network - Network Cloud
```
Comment (jsquire) [inline .github/CODEOWNERS:737]: Please move this to the management library section after L946 and follow the patterns there, updating the path and label entries and including designating service owners for support purposes.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -734,6 +734,9 @@
...
+/sdk/networkcloud/                                                 @Azure/azure-sdk-write-networkcloud
```
Comment (jsquire) [inline .github/CODEOWNERS:738]: Team won't be recognized by the linter for 24 hours after creation.  Assuming that it has the necessary configuration, we'll just need to rerun that step tomorrow.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1003,6 +1003,12 @@
 # ServiceLabel: %Network - Mobile %Mgmt
 # ServiceOwners:                                                   @ArthurMa1978
 
+# PRLabel: %NetworkCloud
+/sdk/networkcloud/                                                 @Azure/azure-sdk-write-networkcloud
```
Comment (jsquire) [inline .github/CODEOWNERS:1007]:
```suggestion
/sdk/networkcloud/Azure.ResourceManager.*/                         @Azure/azure-sdk-write-networkcloud
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1003,6 +1003,12 @@
 # ServiceLabel: %Network - Mobile %Mgmt
 # ServiceOwners:                                                   @ArthurMa1978
 
+# PRLabel: %NetworkCloud
+/sdk/networkcloud/Azure.ResourceManager.*/                         @Azure/azure-sdk-write-networkcloud
+
+# ServiceLabel: %NetworkCloud %Mgmt
+# ServiceOwners:                                                   @Azure/azure-sdk-write-networkcloud
```
Comment (jsquire) [inline .github/CODEOWNERS:1010]:
```suggestion
# PRLabel: %Network - Cloud
/sdk/networkcloud/Azure.ResourceManager.*/                         @Azure/azure-sdk-write-networkcloud

# ServiceLabel: %Network - Cloud %Mgmt
# ServiceOwners:                                                   @Azure/azure-sdk-write-networkcloud
```

**Comment by jsquire** [comment]
Added reformatted label to match repository naming pattern for network subcategories.   Will need to kick the linter again tomorrow after it refreshes the label list.

**Comment by jsquire** [comment]
@JimSuplizio:  The label that is being flagged by the linter exists in the repository with exact capitalization and spacing.  Any idea why the linter is rejecting it?

**Comment by JimSuplizio** [comment]
> @JimSuplizio: The label that is being flagged by the linter exists in the repository with exact capitalization and spacing. Any idea why the linter is rejecting it?

@jsquire if you looked at the date on the failed run it was from yesterday. I reran it today and it passed. If the label was added yesterday it wouldn't be in the repo/label cache until this morning and that would explain this.

**Comment by jsquire** [comment]
> We are working to get the NetworkCloud PR Label registered.

This will be `Network - Cloud` and has been created in the .NET repository.    There's a number of `Network - ` labels, and we'll want this to follow that pattern.

//cc: @ronniegeraghty

**Comment by jsquire** [comment]
> > @JimSuplizio: The label that is being flagged by the linter exists in the repository with exact capitalization and spacing. Any idea why the linter is rejecting it?
> 
> @jsquire if you looked at the date on the failed run it was from yesterday. I reran it today and it passed. If the label was added yesterday it wouldn't be in the repo/label cache until this morning and that would explain this.

Dang it.  Sorry.  I swore that I kicked that over again this morning.

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
@cn0238: Repository labels are chosen by the Azure SDK team and generally are required to follow our patterns.  @ronniegeraghty has final approval on label names and will request any adjustments based on branding concerns and likewise.

**Pull request merged**